### ISSUE TITLE
(Request) support for sfrom tool

### ISSUE THREAD

**Issue by gingerbeardman** [daPhie79/hakchi2#8]
http://darkakuma.z-net.us/p/sfromtool.html

also see https://github.com/daPhie79/hakchi2/issues/6

**Comment by daPhie79** [comment]
indeed. that is planned as well. /u/DarkAkuma is already aware, and we agreed on a few details. i'm waiting on an updated version that will be able to cooperate with hakchi2!
